setup/win/mmex_x64_setup.iss

```diff
@@ -77,29 +80,34 @@ ArchitecturesAllowed=x64
 ArchitecturesInstallIn64BitMode=x64 
 
 [Languages]
-Name: english; MessagesFile: compiler:Default.isl; 
-Name: russian; MessagesFile: compiler:Languages\Russian.isl;
-Name: brazilianportuguese; MessagesFile: compiler:Languages\BrazilianPortuguese.isl; 
-Name: catalan; MessagesFile: compiler:Languages\Catalan.isl; 
-Name: czech; MessagesFile: compiler:Languages\Czech.isl; 
-Name: danish; MessagesFile: compiler:Languages\Danish.isl; 
-Name: dutch; MessagesFile: compiler:Languages\Dutch.isl; 
-Name: finnish; MessagesFile: compiler:Languages\Finnish.isl; 
-Name: french; MessagesFile: compiler:Languages\French.isl; 
-Name: german; MessagesFile: compiler:Languages\German.isl; 
-Name: greek; MessagesFile: compiler:Languages\Greek.isl; 
-Name: hebrew; MessagesFile: compiler:Languages\Hebrew.isl; 
-Name: hungarian; MessagesFile: compiler:Languages\Hungarian.isl; 
-Name: italian; MessagesFile: compiler:Languages\Italian.isl; 
-Name: japanese; MessagesFile: compiler:Languages\Japanese.isl; 
-Name: norwegian; MessagesFile: compiler:Languages\Norwegian.isl; 
-Name: polish; MessagesFile: compiler:Languages\Polish.isl; 
-Name: portuguese; MessagesFile: compiler:Languages\Portuguese.isl; 
-Name: serbiancyrillic; MessagesFile: compiler:Languages\SerbianCyrillic.isl; 
-Name: serbianlatin; MessagesFile: compiler:Languages\SerbianLatin.isl; 
-Name: slovenian; MessagesFile: compiler:Languages\Slovenian.isl; 
-Name: spanish; MessagesFile: compiler:Languages\Spanish.isl; 
-Name: ukrainian; MessagesFile: compiler:Languages\Ukrainian.isl; 
+Name: "english"; MessagesFile: "compiler:Default.isl"
+Name: "basque"; MessagesFile: "compiler:Languages\Basque.isl"
```

Comment:
The languages here are the ones that InnoSetup provides at startup. Any missing ones have not been provided.
5 extra languages have been provided.